server/datastore/mysql/software.go

```diff
@@ -574,10 +576,28 @@ func addCPEForSoftwareDB(ctx context.Context, exec sqlx.ExecerContext, software
 	return uint(id), nil
 }
 
-func (ds *Datastore) AllCPEs(ctx context.Context) ([]string, error) {
-	sql := `SELECT cpe FROM software_cpe`
+func (ds *Datastore) AllCPEs(ctx context.Context, excludedPlatforms []string) ([]string, error) {
```

Comment:
Looks good, in the future once we start supporting specifics flows for specific platforms in a more defined way we can change this to `AllCPEs(ctx context.Context, platforms []string)` (basically, get all the CPEs for the given platforms).